test.py

```diff
@@ -0,0 +1,51 @@
+from os import system
+import json
+
+
+def inputize(d : dict) -> str:
+    return '\'' + json.dumps(d) + '\''
+
+
+tmpfile = 'tmp0801371794747427242472.txt'
+deployed = {}
+
+
+def deploy(file : str):
+    system(f'./evm2near {file} -o test.wasm -b near')
+    print(f'Contract {file} compiled succesfully')
+    system(f'near --networkId testnet dev-deploy test.wasm >{tmpfile}')
+    with open(tmpfile, 'r') as f:
+        s = f.readline()
+        accountId = s[s.find('dev'):s.find(',')]
+    print(f'Contract {file} deployed to testnet with accountId={accountId}')
+    deployed[file] = accountId
+
+
+def TestOne(file : str, method : str, input : dict, expected_output : str):
+    if file not in deployed:
+        deploy(file)
+    accountId = deployed[file]
+    system(f'near call --account-id {accountId} {accountId}' + f' {method}  {inputize(input)}' + f' >{tmpfile}')
+    with open(tmpfile, 'r') as f:

```

Comment:
I assume there is a way in Python to capture the output from a command instead of writing the output to a file then reading that file.
For example, I know for certain the `subprocess` module has this functionality.